lib/ansible/module_utils/facts.py

```diff
@@ -169,10 +169,13 @@ class Facts(object):
                  { 'path' : '/usr/local/sbin/pkg',  'name' : 'pkgng' },
                 ]
 
-    def __init__(self, module, load_on_init=True):
+    def __init__(self, module, load_on_init=True, cached_facts={}):
```

Comment:
pass None, you really want to avoid [] and {} as initializers in python functions